src/polymer-cli.ts

```diff
@@ -103,6 +104,17 @@ export class PolymerCli {
       }
     }
 
+    // This is a quick fix to make sure that "webcomponentsjs" files are
+    // included in every build, since some are imported dynamically in a way
+    // that our analyzer cannot detect.
+    // TODO(fks) 03-07-2017: Remove/refactor when we have a better plan for
+    // support (either here or inside of polymer-project-config).
+    this.defaultConfigOptions = this.defaultConfigOptions || {};
+    this.defaultConfigOptions.extraDependencies =
+        this.defaultConfigOptions.extraDependencies || [];
+    this.defaultConfigOptions.extraDependencies.push(
+        `bower_components${pathSeperator}webcomponentsjs${pathSeperator}*.js`);
```

Comment:
not needed, this gets passed to the PolymerProject constructor as if it were provided in a polymer.json.